gammapy/modeling/fit.py

```diff
@@ -359,23 +359,63 @@ def stat_profile(
 
         return {"values": values, "stat": np.array(stats)}
 
-    def stat_contour(self):
+    def stat_profile_2D(
```

Comment:
To me a "2d profile" sounds a bit strange, maybe we call it e.g. `stat_surface()` instead?